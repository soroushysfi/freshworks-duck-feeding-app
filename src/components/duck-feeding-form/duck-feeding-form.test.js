import { shallow, mount, render } from 'enzyme'
import DuckFeedingForm from './duck-feeding-form'



it('expect to match DuckFeedingForm snapshot', () => {
    expect(shallow(<DuckFeedingForm />)).toMatchSnapshot()

})