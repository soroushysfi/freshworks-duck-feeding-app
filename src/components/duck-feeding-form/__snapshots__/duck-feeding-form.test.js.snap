// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to match DuckFeedingForm snapshot 1`] = `
<form
  className="login-form box-format"
  onSubmit={[Function]}
>
  <h1>
    Submit your duck feeding information!
  </h1>
  <div
    className="display-row"
  >
    <CustomInputComponent
      label="Feeding Time"
      name="feedingTime"
      onChange={[Function]}
      placeholder="When did you feed the ducks"
      required={true}
      type="date"
      value=""
    />
    <CustomInputComponent
      label="Food Type"
      name="foodType"
      onChange={[Function]}
      placeholder="Like vegetable"
      required={true}
      type="text"
      value=""
    />
  </div>
  <div
    className="display-row"
  >
    <CustomInputComponent
      label="Food Kind"
      name="foodKind"
      onChange={[Function]}
      placeholder="Like broccoli"
      required={true}
      type="text"
      value=""
    />
    <CustomInputComponent
      label="Duck Count"
      name="duckCount"
      onChange={[Function]}
      placeholder="How many ducks?"
      required={true}
      type="number"
      value=""
    />
  </div>
  <div
    className="display-row"
  >
    <CustomInputComponent
      label="Food Amount"
      name="foodAmount"
      onChange={[Function]}
      placeholder="How many grams?"
      required={true}
      type="number"
      value=""
    />
  </div>
  <div
    className="display-row"
  >
    <CustomInputComponent
      label="Re-occurring Feeding"
      name="reOccurring"
      onChange={[Function]}
      type="checkbox"
      value={false}
    />
  </div>
  <button
    className="btn"
    disabled={false}
    type="submit"
  >
    Submit
  </button>
</form>
`;
